dotcom-rendering/src/web/components/Contributor.tsx

```diff
@@ -15,25 +16,59 @@ import TwitterIcon from '../../static/icons/twitter.svg';
 import { interactiveLegacyClasses } from '../layouts/lib/interactiveLegacyStyling';
 import { decidePalette } from '../lib/decidePalette';
 
-const twitterHandleStyles = (palette: Palette) => css`
-	${textSans.xxsmall()};
-	font-weight: bold;
+const defaultTwitterHandleColour = (palette: Palette) => css`
```

Comment:
This is subject so take it or leave it but I find nesting css variables can be confusing. I am a fan of keeping things simple and having all the css code used by a variable declaration contained inside itself. And then you put any logic in `<div css={myLogic && simpleCss}>`